.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 dist/
 build/
 sandbox.js
+.DS_Store
```

Comment:
This should really live in your [global __.gitignore__][1].


[1]: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore